tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -311,7 +311,7 @@ Other options:
                            * inferredParser (string | null) - name of parser inferred from file path
   -h, --help <flag>        Show CLI usage, or details about the given flag.
                            Example: --help write
-  --ignore-unknown         Ignore unknown files.
+  -i, --ignore-unknown     Ignore unknown files.
```

Comment:
`-i` in the most tools alias to `--ignore`, i think we should keep same behavior

Reply:
Are you suggestion use another alias?

Reply:
Yes, using other flag, in many tools `-i` means `--ignore` (`eslint`/`stylelint`/etc), so it can be misleading

Reply:
`-u`?

Reply:
I think better `-iu` (by first letters of words)

Reply:
I'll see if we support `-iu` first

Reply:
```text
node bin/prettier . -iu -l
[warn] Ignored unknown option -i. Did you mean -_?
[warn] Ignored unknown option -u. Did you mean -_?
```

:smile:

Reply:
hm, strange, do we use `meow`?

Reply:
https://github.com/prettier/prettier/blob/6793811da2d2304b711da08833e3da704074cbbf/src/cli/util.js#L821

Reply:
:disappointed:

Reply:
Just a guess: Some CLI libraries parse into `{ option1: true, option2: "foo", _: ["rest", "of", "args"] }`. Maybe we do too and mistakenly use the `_` when providing suggestions?

Reply:
I have to say the args parsing is a little hard to understand, I did a little digging when I try to do this PR https://github.com/prettier/prettier/pull/8831

Reply:
I had a little digging, seems `minimist` don't support `-iu`, it always split them https://github.com/substack/minimist/blob/aeb3e27dae0412de5c0494e9563a5f10c82cc7a9/index.js#L140, and most important, `iu` flag is not setting to correct `boolean`.

Let's go with `-u`

Reply:
:+1: